app.js

```diff
@@ -1,56 +1,251 @@
-  new Product ('breakfast'),
-  new Product ('bubblegum'),
-  new Product ('chair'),
-  new Product ('cthulhu'),
-  new Product ('dog-duck'),
-  new Product ('dragon'),
-  new Product ('pen'),
-  new Product ('pet-sweep'),
-  new Product ('reset'),
-  new Product ('scissors'),
-  new Product ('shark'),
-  new Product ('sweep'),
-  new Product ('tauntaun'),
-  new Product ('unicorn'),
-  new Product ('water-can'),
-  new Product ('wine-glass'),
-];
+    let item2 = randomNumberArray.shift();
+    let image2 = document.getElementById('item2');
+    image2.src = itemsArray[item2].src;
+    image2.alt = itemsArray[item2].name;
 
-console.log(allProducts);
+    let item3 = randomNumberArray.shift();
+    let image3 = document.getElementById('item3');
+    image3.src = itemsArray[item3].src;
+    image3.alt = itemsArray[item3].name;
+
+    //counting how many times the items were viewed
+    itemsArray[item1].views++;
+    itemsArray[item2].views++;
+    itemsArray[item3].views++;
+  }
+}
 
-function randomImage() {
-  let result = Math.floor(Math.random() * allProducts.length);
-  return allProducts[result];
 
+//event handler for items and number of likes they get
+function handleLikes(event) {
+  counter++;
+  for (let i = 0; i < itemsArray.length; i++) {
+    if (event.target.alt === itemsArray[i].name) {
+      itemsArray[i].likes++;
+      break;
+    }
+  }
+  // when the counter reaches maxClicks, the results render
+  if (counter === maxClicks) {
+    images.removeEventListener('click', handleLikes);
+    renderResults();
+  } else {
+    // random items keep rendering until counter === maxClicks
+    renderItems();
+  }
 }
 
-let image = randomImage();
-console.log(image);
 
+// stores results of each voting round in localStorage
+function storeItems(arr) {
+  // KEY name is 'saveItems'
+  // stringifies objects to put into localStorage
+  let resultsData = JSON.stringify(arr);
+  localStorage.setItem('saveItems', resultsData);
+}
 
-//<img></img>>
 
-let img = document.createElement('img');
+// retrieves previous voting results from localStorage
+function renderData() {
+  let findData = localStorage.getItem('saveItems');
+  // "un-stringify" results from localStorage which turn into 'POJO' (plain old JS object)
+  let showData = JSON.parse(findData);
+  //check to see if there is data in localStorage with if statement
+  if (findData) {
+    // re-assemble properties of previous results
+    for (let order of showData) {
+      let views = order.views;
+      let likes = order.likes;
+      let name = order.name;
+      let fileExtension = order.fileExtension;
+      let src = order.src;
+      // POJOs get re-instantiated by going through createItems(); again
+      createItems(views, likes, name, fileExtension, src);
+    }
+    // if no previous results are found in localStorage, run page from the beginning (initial run)
+  } if (!findData) {
+    // called to create the first instances of Items objects
+    initialResults();
```

Comment:
This is an excellent way to render initial data! It's very readable. Great work!